cogs/general.py

```diff
@@ -154,8 +160,11 @@ def __init__(self, bot):
     @commands.command()
     async def lmgtfy(self, *, search_terms : str):
         """Creates a lmgtfy link"""
-        search_terms = escape_mass_mentions(search_terms.replace("+","%2B").replace(" ", "+"))
-        await self.bot.say("https://lmgtfy.com/?q={}".format(search_terms))
+        if len(search_terms) > 1000:
+            await self.bot.say("I think you know why I won't allow this...")
```

Comment:
At least put in brackets at the end `(Too long)` or something.

Reply:
And also, does google have a limit of 1000 characters for searches?

Reply:
I think similar to other PRs, this is to address Discord message content limits.